backend/GameManager.py

```diff
@@ -1,17 +1,41 @@
 # import pygame as pg
 from backend.Board import Board
-from backend.Cell import CellType
-from backend.Piece import Pieces, Turn
+from backend.Piece import Pieces
+from backend.Cell import Color
 
 
 class GameManager:
     def __init__(self, size, pieces):
         self.board = Board(size)
         self.pieces = Pieces(pieces)
-        self.turn = Turn.WHITE
+        self.turn = Color.WHITE
+        self.selected_piece = None
+        self.open_moves = self.board.get_valid_moves()
+        
+        # Mill variables
+        self.mills = [
```

Comment:
Same thing here. We need a programmatic way of determining mill.

Logic: After piece is placed do the other 2 positions that share an axis have the current player pieces in them?

Reply:
same i explained above